### PR TITLE
[DO NOT MERGE] Update dependencies to change to logrus 1.0.0

### PR DESCRIPTION
**Pull request by mrunalp**
Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

**Comment by runcom** [comment]
LGTM if tests are ok

**Comment by mrunalp** [comment]
The test failure doesn't look related. Could you check?

**Comment by runcom** [comment]
yup, restarted

**Comment by rhatdan** [comment]
Still failing.

**Comment by mrunalp** [comment]
```
/Users/travis/build/projectatomic/skopeo/_gopath/src/github.com/projectatomic/skopeo/hack/make/validate-lint: line 14: golint: command not found
make: *** [validate-local] Error 127
```

**Comment by runcom** [comment]
@mtrmac have you ever seen that? first time here :/

**Comment by mtrmac** [comment]
@runcom I haven’t yet, but I will be setting up a macOS environment anyway, so I’ll take a look.

**Comment by mtrmac** [comment]
LGTM pending tests.

**Comment by mtrmac** [comment]
LGTM pending tests.

**Comment by runcom** [comment]
Needs a rebase?

**Comment by runcom** [comment]
Yup, this just needs a rebase from travis's failure

**Comment by mrunalp** [comment]
Rebased.

File: vendor.conf [hunk 1]
```diff
@@ -1,9 +1,9 @@
 github.com/urfave/cli v1.17.0
-github.com/containers/image master
+github.com/containers/image update_rename_logrus https://github.com/mrunalp/image
```
Comment (runcom) [inline vendor.conf]: @mrunalp can you put master back and revendor now that I've merged the PR in c/image?

**Pull request merged**